plugins/dune-rules/src/qcheck_stm.ml

```diff
@@ -0,0 +1,154 @@
+type config = {
+  interface_file : string;
+  init_function : string;
+  sut_type : string;
+  ocaml_output : string;
+  include_ : string option;
+  package_name : string option;
+  dune_output : string option;
+}
+
+type plugin = Dune | QCheckSTM
+
+open Fmt
+
+let quote ppf s = pf ppf "\"%s\"" s
```

Comment:
If _think_ dune is using OCaml syntax. If indeed so, `pf ppf "%S" s` would be more robust there.

Reply:
You're right :-)